pvlib/spectrum/magnus_tetens.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+
+
+def magnus_tetens_aekr(temperature, dewpoint, A=6.112, B=17.62, C=243.12):
+    """
+    Calculate relative humidity using Magnus equation with AEKR coefficients.
+    This function was used by First Solar in creating their spectral model
+    and is therefore relevant to the first solar spectral model in pvlib.
+    Default magnus equation coefficients are from [2].
+
+    Parameters
+    ----------
+    temperature : pd.Series
+        Air temperature in degrees Celsius
+    dewpoint : pd.Series
+        Dewpoint temperature in degrees Celsius
+    A: float
+        Magnus equation coefficient A
+    B: float
+        Magnus equation coefficient B
+    C: float
+        Magnus equation coefficient C
```

Comment:
Mild preference to specify these as a single `coeff=(6.112, 17.62, 243.12)` parameter, I think